src/fasta-recs.py

```diff
@@ -1,18 +1,32 @@
-import argparse
+import sys
 
+def read_fasta():
+    # load input:
+    inFile = sys.argv[1]
+    with open(inFile,'r') as j:
+        lines = j.readlines()
+    
+    # make fasta parser:
+    record_list = []
+    header = ''
+    sequence = ''
+    for line in lines:
+        line = line.strip()
+        if line.startswith('>'):
+            if header != "":
+                record_list.append([header, sequence])
```

Comment:
Use a tuple, `(header, sequence)` instead of a list `[header, sequence]`. It is safer in many settings and it is also less likely to confuse an average Python programmer.